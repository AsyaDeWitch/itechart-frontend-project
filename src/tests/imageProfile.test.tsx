import { /* render, */ unmountComponentAtNode } from "react-dom";
import renderer from "react-test-renderer";
import ImageProfile from "../components/users/elements/imageProfile/imageProfile";

let container: Element;

beforeEach(() => {
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container?.remove();
});

describe("Image profile", () => {
  test("Snapshot test should be the same with previous version", () => {
    const mockImage64 = "";

    const component = renderer.create(<ImageProfile image64={mockImage64} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
