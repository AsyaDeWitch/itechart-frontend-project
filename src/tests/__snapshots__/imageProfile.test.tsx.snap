// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image profile Snapshot test should be the same with previous version 1`] = `
<div
  className="image-profile__container"
>
  <div
    className="image-profile"
  >
    <img
      alt="Profile"
      className="image-profile__image"
      src=""
    />
  </div>
</div>
`;
